### DISCUSSION TITLE
Singleton resource panels / custom additional stubs

### DISCUSSION THREAD

**Discussion by bchubb-web** [filamentphp/filament#13820] [Ideas]
I wouldn't be surprised if this has been brought up before, but i was wondering if a native implementation of a resource that manages a single instance of a model was on the roadmap for v4.x, it is something i implement myself in pure laravel projects and assumed filament would provide an option for this like --single

I find this useful for global config or resources that represent something specific like a legal page on a site.

If not, a more flexible option would be to add a --type flag (like laravel's make:x commands) to specify a custom stub template, so users could have resource.foo.sub, rather than overwriting the published resource.stub

**Comment by ignacio-dev** [comment]
Agreed! Such an important part of a cms in my opinion. Hate having to do workarounds, which work fine with simple stuff, but gets unnecesarily complex as soon as you introduce things like translations.

**Reply by bchubb-web** [reply]
by translations do you mean translated columns? I havent touched that side of laravel/filament, what differences would there be compared to a regular resource in this example?
